tests/ApiTest.php

```diff
@@ -242,7 +266,8 @@ function () {
 	 * Test the api_login() function without any login.
 	 * @return void
 	 * @runInSeparateProcess
-	 * @expectedException Friendica\Network\HTTPException\UnauthorizedException
+	 *
+	 * @expectedException \Friendica\Network\HTTPException\UnauthorizedException
```

Comment:
I think both forms are valid (but I don't have a strong preference).

Reply:
PHPUnit didn't recognize the old form. When I changed the form, I got different expected exceptions (see my second commit)

Reply:
What do you mean it didn't recognize the old form? Tests are currently working fine on Travis and phpunit correctly catches these exceptions.

I think you got different expected exceptions because you changed it from `ForbiddenException` to `BadRequestException` in your first commit [here](https://github.com/friendica/friendica/pull/5350/commits/48012ed5937d76a3ecb0abe75113de717507df9e#diff-1fb4c51895a21397cab0634af7b5e2edR1260).

But again, I don't mind using this new form if you think it is easier to read.

Reply:
Ah damn, you're right :-(